docs/scenarios.md

```diff
@@ -0,0 +1,55 @@
+### Scenarios
+Kraken currently supports pod and node based scenarios. 
+
+
+#### Pod chaos scenarios
+Following are the components of Kubernetes/OpenShift for which a basic chaos scenario config exists today. Adding a new pod based scenario is as simple as adding a new config under scenarios directory and defining it in the config.
+
+Component                | Description                                                                                        | Working
+------------------------ | ---------------------------------------------------------------------------------------------------| ------------------------- |
+Etcd                     | Kills a single/multiple etcd replicas for the specified number of times in a loop                  | :heavy_check_mark:        |
+Kube ApiServer           | Kills a single/multiple kube-apiserver replicas for the specified number of times in a loop        | :heavy_check_mark:        |
+ApiServer                | Kills a single/multiple apiserver replicas for the specified number of times in a loop             | :heavy_check_mark:        |
+Prometheus               | Kills a single/multiple prometheus replicas for the specified number of times in a loop            | :heavy_check_mark:        |
```

Comment:
Not sure we need to add all scenarios to this table as they are developed, but we also have the Pod RegEx scenario to kill random pods in system or user namespaces based on regular selection.

Reply:
Added the random pod killing scenario to the table.